release-notes/7.0/supported-os.md

```diff
@@ -28,13 +28,13 @@ OS                                    | Version                 | Architectures
 OS                                    | Version               | Architectures     | Lifecycle
 --------------------------------------|-----------------------|-------------------|----------
 [Alpine Linux][Alpine]                | 3.15+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
-[CentOS Linux][CentOS]                | 7                     | x64               | [CentOS][CentOS-lifecycle]
-[CentOS Stream Linux][CentOS]         | 8                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS][CentOS]                      | 7                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS Stream][CentOS]               | 8+                    | x64               | [CentOS][CentOS-lifecycle]
 [Debian][Debian]                      | 10+                   | x64, Arm64, Arm32 | [Debian][Debian-lifecycle]
 [Fedora][Fedora]                      | 36+                   | x64               | [Fedora][Fedora-lifecycle]
 [openSUSE][OpenSUSE]                  | 15+                   | x64               | [OpenSUSE][OpenSUSE-lifecycle]
-[Oracle Linux][Oracle-Linux]          | 7+                    | x64               | [Oracle][Oracle-lifecycle]
-[Red Hat Enterprise Linux][RHEL]      | 7+                    | x64, Arm64        | [Red Hat][RHEL-lifecycle]
+[Oracle Linux][Oracle-Linux]          | 8+                    | x64               | [Oracle][Oracle-lifecycle]
+[Red Hat Enterprise Linux][RHEL]      | 8+                    | x64, Arm64        | [Red Hat][RHEL-lifecycle]
```

Comment:
I think this is a scenario where Red Hat's support and Microsoft's support differ. AFAIK, Microsoft supports .NET 7 on RHEL 7 and later. Red Hat supports .NET 7 on RHEL 8 and later only. Not sure which one this table is supposed to reflect, but "8+" is the intersection of those policies :)

Reply:
According to https://learn.microsoft.com/en-us/dotnet/core/install/linux-rhel#supported-distributions RHEL 7 is only supported in .NET 6, not in .NET 7. I would therefore go with the table from there. So I don't think there's a difference in what Microsoft supports and what RHEL supports here.

.NET 6: RHEL 7+
.NET 7: RHEL 8+

Since RHEL 7 is EOL in 2026 (way after .NET 6 is EOL), it will stay 7+ until .NET 6 is gone and the 6.0 document will not be adjusted.

Reply:
This is an aggregation of the supported distributions together with their next upcoming EOLs that we maintain internally, based on the Microsoft Learn pages and the according `supported-os` pages.
![image](https://user-images.githubusercontent.com/940619/231371873-cffed339-a521-4c90-bcba-f25f6fb178b2.png)
Darker red means there was a change between .NET 6 and .NET 7 support, blue means EOL but according to the documentation still supported (with tickets on https://github.com/dotnet/core/labels/os-support existing).